src/content/reference/gsctl/show-cluster.md

```diff
@@ -79,6 +81,7 @@ The output lines in detail:
 - **AWS account:** (_only on AWS_) If the cluster is running using non-default provider credentials, here we show the AWS account ID
 - **Azure subscription:** (_only on Azure_) If the cluster is running using non-default provider credentials, here we show the subscription ID
 - **Azure tenant:** (_only on Azure_) If the cluster is running using non-default provider credentials, here we show the tenant ID
+- **Labels** (_only on AWS_) user defined labels. Only available for clusters with node pools
```

Comment:
Actually, only on AWS for clusters using releases >v10.x.x.

Also, `Only available for clusters with node pools` sounds a bit confusing. A v11.x.x cluster can have labels even without having a node pool.